tests/director/tests/sensitive_data_strip_test.go

```diff
@@ -0,0 +1,257 @@
+	appAuth := fixtures.RequestClientCredentialsForApplication(t, context.Background(), dexGraphQLClient, tenantId, app.ID)
+	appOauthCredentialData, ok := appAuth.Auth.Credential.(*graphql.OAuthCredentialData)
+	require.True(t, ok)
+	require.NotEmpty(t, appOauthCredentialData.ClientSecret)
+	require.NotEmpty(t, appOauthCredentialData.ClientID)
+	applicationOAuthGraphQLClient := gqlClient(t, appOauthCredentialData, token.ApplicationScopes)
+
+	// register integration system
+	t.Log(fmt.Sprintf("Registering integration system %q", intSysName))
+	integrationSystem := fixtures.RegisterIntegrationSystem(t, ctx, dexGraphQLClient, tenantId, intSysName)
+	defer fixtures.UnregisterIntegrationSystem(t, ctx, dexGraphQLClient, tenantId, integrationSystem.ID)
+
+	// register integration system oauth client
+	intSysAuth := fixtures.RequestClientCredentialsForIntegrationSystem(t, context.Background(), dexGraphQLClient, tenantId, integrationSystem.ID)
+	intSysOauthCredentialData, ok := intSysAuth.Auth.Credential.(*graphql.OAuthCredentialData)
+	require.True(t, ok)
+	require.NotEmpty(t, intSysOauthCredentialData.ClientSecret)
+	require.NotEmpty(t, intSysOauthCredentialData.ClientID)
+	intSystemOAuthGraphQLClient := gqlClient(t, intSysOauthCredentialData, token.IntegrationSystemScopes)
+
+	// assign runtime and app to the same scenario
+	scenarios := []string{conf.DefaultScenario, "test-scenario"}
+	fixtures.UpdateScenariosLabelDefinitionWithinTenant(t, ctx, dexGraphQLClient, tenantId, scenarios)
+	defer fixtures.UpdateScenariosLabelDefinitionWithinTenant(t, ctx, dexGraphQLClient, tenantId, scenarios[:1])
+
+	// set application scenarios label
+	fixtures.SetApplicationLabel(t, ctx, dexGraphQLClient, app.ID, ScenariosLabel, scenarios[1:])
+	defer fixtures.SetApplicationLabel(t, ctx, dexGraphQLClient, app.ID, ScenariosLabel, scenarios[:1])
+
+	// set runtime scenarios label
+	fixtures.SetRuntimeLabel(t, ctx, dexGraphQLClient, tenantId, runtime.ID, ScenariosLabel, scenarios[1:])
+	defer fixtures.SetRuntimeLabel(t, ctx, dexGraphQLClient, tenantId, runtime.ID, ScenariosLabel, scenarios[:1])
+
+	// create bundle instance auth
+	t.Log(fmt.Sprintf("Creating bundle instance auths %q with bundle with APIDefinition and ", appName))
+	instanceAuth := fixtures.CreateBundleInstanceAuth(t, ctx, dexGraphQLClient, bndl.ID)
+	require.NotNil(t, instanceAuth)
+
+	t.Run("Application access", func(t *testing.T) {
+		type accessRequired struct {
+			appWebhooks               bool
+			appAuths                  bool
+			bundleInstanceAuth        bool
+			bundleInstanceAuths       bool
+			bundleDefaultInstanceAuth bool
+			documentFetchRequest      bool
+			eventSpecFetchRequest     bool
+			apiSpecFetchRequest       bool
+		}
+		type testCase struct {
+			name              string
+			consumer          *gcli.Client
+			fieldExpectations accessRequired
+		}
+		testCases := []testCase{
+			{
+				name:     "Runtime consumer",
```

Comment:
we also have int system consumer

Reply:
line 130 :) no fields should be accessible